crmsh/bootstrap.py

```diff
@@ -2417,4 +2406,52 @@ def get_stonith_timeout_generally_expected():
         return None
 
     return STONITH_TIMEOUT_DEFAULT + corosync.token_and_consensus_timeout()
+
+
+def adjust_pcmk_delay():
+    """
+    For each fence agent,
+    add parameter pcmk_delay_max when cluster is two-node cluster without qdevice
+    else remove pcmk_delay_max
+    """
+    cib_factory.refresh()
+
+    if utils.is_2node_cluster_without_qdevice():
+        for res in cib_factory.fence_id_list_without_pcmk_delay():
+            cmd = "crm resource param {} set pcmk_delay_max {}s".format(res, PCMK_DELAY_MAX)
+            utils.get_stdout_or_raise_error(cmd)
+            logger.debug("Add parameter 'pcmk_delay_max={}s' for resource '{}'".format(PCMK_DELAY_MAX, res))
+    else:
```

Comment:
Minor argument, the "else" branch contributes nothing to the cluster behavior, deleting it could save 4 lines of code ;)

Reply:
No harm to keep it, else I need to adjust the test cases:)